python/paddle/fleet/base/distributed_strategy.py

```diff
@@ -378,6 +378,30 @@ def fuse_all_reduce_ops(self, flag):
         else:
             print("WARNING: fuse_all_reduce_ops should have value of bool type")
 
+    @property
+    def fuse_grad_size_in_MB(self):
+        return self.strategy.fuse_grad_size_in_MB
+
+    @fuse_grad_size_in_MB.setter
+    def fuse_grad_size_in_MB(self, value):
+        if isinstance(value, int):
+            self.strategy.fuse_grad_size_in_MB = value
+        else:
+            print("WARNING: fuse_grad_size_in_MB should have value of int type")
+
+    @property
+    def _fuse_grad_size_in_TFLOPS(self):
+        return self.strategy.fuse_grad_size_in_TFLOPS
+
+    @_fuse_grad_size_in_TFLOPS.setter
+    def _fuse_grad_size_in_TFLOPS(self, value):
```

Comment:
Same as above?

Reply:
Oh, this should be float, I will fix it.